config/routes/admin.rb

```diff
@@ -149,8 +149,9 @@
   # Search
   namespace :search do
     resources :users, only: :index
+    resources :purchases, only: :index
   end
-  get "/search_purchases", to: "search#purchases", as: :search_purchases
+  get "/search_purchases", to: "search#purchases"
```

Comment:
Keeping the legacy endpoint around for now as it seems like that's what we've been doing elsewhere, but can remove the controller here if you prefer